### ISSUE TITLE
Add support for military time

### ISSUE THREAD

**Issue by jdiefen** [RikkiGibson/Corvallis-Bus-Server#13]
Currently the times are sent to the clients in 12-hour time, regardless of the client's device settings.

**Comment by cartermp** [comment]
Two reasonable approaches:
1. Send a flag to the server indicating military time
2. Deal with the time on the app

I think (2) is the better option, since you'd need to detect device settings anyways.

**Comment by RikkiGibson** [comment]
(1) will have less of an impact on existing apps, because the server has basically complete control over time: https://corvallisb.us/api/arrivals-summary/10308,14237

It was useful to dedupe a bit of business logic, but clearly has its drawbacks.
I'm thinking of just adding a time format string as an API parameter and using the existing hard-coded one in the server as a default. Then the client will need to decide which time format string to send based on device settings.

Some plumbing around in iOS and Android APIs may confirm whether it's sensible to do this, or to just send a darn boolean. It's basically a question of "how hyped are you about copy/pasting format strings between projects?"

**Comment by RikkiGibson** [comment]
The combination of Connexionz providing integer "minutes from now" values and the schedule consisting of DateTimeOffset made it a big pain in the behind to provide proper time values from the server. The "minutes from now" value can turn into a different number when you convert it to an absolute time and then back to a relative time, and since these values are really loose anyway, I'd like to be able to make sure the server delivers the same number to the client that it got from Connexionz.

This basically means that each "arrival time" value emitted by the server can either be a Connexionz estimate or a scheduled timestamp, so you're looking at either a heterogeneous, ordered list of arrival times, or 2 homogeneous lists that people will have to correctly interleave themselves--which requires the reference timestamp the server would have used.

Basically, I just refreshed myself on why I pushed all the time difficulty to the server and just returned arrival time strings for the UI.. The solution for 24 hour time is probably to accept a date format string as a query parameter.

It looks like such a format string can be created on iOS like so: https://stackoverflow.com/a/16521905/2855742

Probably all the non-US folks who use the service would appreciate getting dates in this more familiar form.
